sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProducer.cs

```diff
@@ -4,10 +4,14 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
+using System.Diagnostics;
 using System.Globalization;
 using System.Threading;
 using System.Threading.Tasks;
+using Azure.Core.Pipeline;
 using Azure.Messaging.EventHubs.Core;
+using Azure.Messaging.EventHubs.Diagnostics;
+using Microsoft.Azure.Amqp;
```

Comment:
This we need to figure out how to remove a dependency on.   This class should be agnostic to the transport and not bound to AMQP.

Reply:
I may be overlooking it, but I don't see where the AMQP reference comes into play at all?  Is this just a leftover from working through things?

Reply:
Was a leftover from some experimentation.